lib/common.go

```diff
@@ -59,7 +59,7 @@ type TaskStatus struct {
 	JobId    int
 	Status   Status
 	Password string
-	Progress []byte // State of permutation
+	Progress string // State of permutation
```

Comment:
Sorry, I forgot to check if the master was still working... It makes sense to return the progress in []byte to the master